third_party/open-policy-agent/gatekeeper/helmify/replacements.go

```diff
@@ -0,0 +1,8 @@
+package main
+
+var replacements = map[string]string{
+	`HELMSUBST_DEPLOYMENT_CONTROLLER_MANAGER_CONTAINER_RESOURCES: ""`: `{{- toYaml .Values.resources | nindent 10 }}`,
```

Comment:
since we are setting for controller-manager, let's make sure .Values have is in the correct level. so we can add these for collector, eraser, etc later easier

```suggestion
	`HELMSUBST_DEPLOYMENT_CONTROLLER_MANAGER_CONTAINER_RESOURCES: ""`: `{{- toYaml .Values.controllerManager.resources | nindent 10 }}`,
```

Reply:
this applies to substs below too